command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
+```
+
+# Config 
+```
+rucio config list
+rucio config add
+rucio config remove
+```
+
+# DID 
+```
+rucio did list
+rucio did show
+rucio did remove
+rucio did touch
+rucio did parent
+rucio did attach
+rucio did detach
+rucio did upload
+rucio did download
+rucio did content list
+rucio did content history
+rucio did content close
+rucio did content open
+rucio did metadata list
+rucio did metadata add
+rucio did metadata remove
+rucio did container add
+rucio did dataset add
+```
+
+# Replica 
+```
+rucio replica list
+rucio replica dataset
+rucio replica pfn
+rucio replica tombstone
+rucio replica state suspicious
+rucio replica state quarantine
+rucio replica state bad
+rucio replica state temp-unavailable
+```
+
+# RSE
+```
+rucio rse list
+rucio rse show
+rucio rse remove
+rucio rse add
+rucio rse update
+rucio rse attribute list
+rucio rse attribute add
+rucio rse attribute remove
+rucio rse distance list
+rucio rse distance add
+rucio rse distance remove
+rucio rse distance set
+rucio rse protocol add
+rucio rse protocol remove
+rucio rse limit add
+rucio rse limit remove
+rucio rse qos list
+rucio rse qos add
+rucio rse qos remove
+
+```
+
+# Rule 
+```
+rucio rule add
+rucio rule remove
+rucio rule show
+rucio rule history
+rucio rule update
+rucio rule list
+```
+
+# Scope 
+```
+rucio scope [list]
+rucio scope add
+```
+
+# Subscription
+```
+rucio subscription list
+rucio subscription update
+rucio subscription add
+rucio subscription touch
+
+```
```

Comment:
Likewise, need to have `show` and `remove`. And likewise, this is a feature request for the future.

One thing I never liked about the subscriptions is that the identifier is its combination of account and name. But they have IDs, so that’s what should be used instead. As such, `list` should provide a table with information like ID, name, and state, and `show`, given an ID, should include everything else.

For `touch` I may need to reconsider. Not just the name, but also the placement. The entity being manipulated is the DIDs, not the subscription. I know that this is a one-to-one match to `rucio-admin subscription reevaluate`, but I just need to think about it some more.